### PR TITLE
Added ethereum network statistics UI

### PR DESCRIPTION
**Pull request by lucashenning**
Hey guys, this is awesome work. Thanks a lot for putting this together. Kudos!

I figured you guys could use a UI to make this more tangible and give users the opportunity to see the status of the overall network. Therefore, I added [eth stats](https://github.com/cubedro/eth-netstats) and have each node report to it. Just run `docker-compose up` and go to http://localhost:3000/ 

image

I dockerized eth-stats for this. So, you might want to clone/fork this repo: https://github.com/lucashenning/eth-netstats

Cheers, 
Lucas

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,6 +14,16 @@ services:
       quorum_net:
         ipv4_address: '172.14.0.99'
 
+  eth-stats:
+    build: https://github.com/lucashenning/eth-netstats.git
+    ports:
+      - '3000:3000'
```
Comment (GavinPacini) [inline docker-compose.yml]: Can we change these to double quotes i.e. `"` for all the other ports we have used them.

File: docker-compose.yml [hunk 2]
```diff
@@ -14,6 +14,16 @@ services:
       quorum_net:
         ipv4_address: '172.14.0.99'
 
+  eth-stats:
+    build: https://github.com/lucashenning/eth-netstats.git
```
Comment (GavinPacini) [inline docker-compose.yml]: Awesome stuff, didn't know you could include a git repo directly!

Could you update this to `https://github.com/DeloitteBlockchain/eth-netstats.git` (we forked your great repo, just to have it all under `DeloitteBlockchain` account).

**Review by GavinPacini — CHANGES_REQUESTED** [review]
Really cool stuff Lucas, thanks so much for contributing. Just two small requested changes and then I am happy. :)

**Review by rodrigo-florenciano — COMMENTED** [review]
Great stuff indeed @lucashenning. I think we could improve the logging of the eth-stats by redirecting it to a file. What do you think? We've already mapped a log folder with the logs of the 7 nodes, I think you can place the monitor log there as well.

**Comment by lucashenning** [comment]
Thank you @GavinPacini @rodrigo-florenciano ! I changed repo url and the ports and redirected the output of eth-stats to a log file. Please make sure to pull the latest update from https://github.com/lucashenning/eth-netstats as well. Let me know if you want me to change anything else. 

Thank you guys, I had a lot of fun wrapping my head around your code :+1: Great tool!
Cheers

**Review by GavinPacini — APPROVED** [review]

**Comment by GavinPacini** [comment]
Thanks for those changes @lucashenning , I synced our fork of your ethstats too (thanks for reminding me).

I've approved. @rodrigo-florenciano you can merge once you are happy. Thanks!

**Pull request merged**